flaskbb/utils/populate.py

```diff
@@ -123,15 +123,19 @@ def update_settings_from_fixture(fixture, overwrite_group=False,
 
             if setting is not None:
                 setting_is_different = (
-                    setting.value != settings[1]["value"]
-                    or setting.value_type != settings[1]["value_type"]
-                    or setting.name != settings[1]["name"]
-                    or setting.description != settings[1]["description"]
-                    or setting.extra != settings[1].get("extra", "")
-                    or setting.settingsgroup != group.key
+                    setting.value != settings[1]["value"] or
```

Comment:
So this is really annoying, black wants to put these on the beginning of the next line but Flake8 wants them here. I might file an issue against black on this since I'm siding with Flake8 on this one. I've seen black do something other weirdo stuff as well.

Reply:
That behaviour of flake8 was apparently based on an older version of PEP8 and has since been corrected: https://github.com/PyCQA/pycodestyle/pull/502

This isn't a bug in black - PEP8 recommends breaking before operators, though permits both styles: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

In the Flask codebase, line breaks are consistently put before operators, so I would suggest we go with that too, for consistency and stuff.

Reply:
Interesting. Looks like flake8 3.5.0 is too old to include that version of pycodestyle (it's in 2.4.0 and flake8 3.5.0 explicitly declares its dependency as `>=2.0.0,<2.4.0`, it was only a month ago that the pycode style was bumped to include 2.4.0. I'll disable the rule and reformat.